src/components/Table/Table.tsx

```diff
@@ -0,0 +1,76 @@
+import { Table, Typography, Input, Select, Row, Col } from "antd";
+import { Content } from "antd/es/layout/layout";
+import React, { useState } from "react";
+import styles from "./Table.module.scss";
+
+interface inputProps {
+  columns: { title: string; dataIndex: string; key: string }[];
+  data: {}[];
+  title: string;
+  sortByOptions: string[];
+}
+
+const { Text } = Typography;
+const { Search } = Input;
+const { Option } = Select;
+
+const SimpleTable: React.FC<inputProps> = ({
+  columns,
+  data,
+  title,
+  sortByOptions,
+}) => {
+  const [filteredData, setFilteredData] = useState(data);
+
+  const handleSearch = (value: string) => {
```

Comment:
what if i don't want want to sort initially ? will the data be shown ?

Reply:
yes data is being shown initially before sorting as well in storybook file, further testing can be done in actual projects when we have more data.